sdk-libs/macros/MACRO_REFACTOR_V2.md

```
@@ -0,0 +1,642 @@
+│  └────────────┬────────────┘         │      bump,                                  │   │
+│               │                      │    )]                                       │   │
+│               │  Scans for           │    #[compressible]                          │   │
+│               │  #[compressible]     │    pub user: Account<'info, UserRecord>,    │   │
+│               │  fields              │                                             │   │
+│               │                      │    #[account(seeds = [b"vault", cmint...],  │   │
+│               │                      │              bump)]                         │   │
+│               │                      │    #[compressible_token(Vault, authority=.)]│   │
+│               │                      │    pub vault: UncheckedAccount<'info>,      │   │
+│               │                      │  }                                          │   │
+│               │                      └───────────────────┬─────────────────────────┘   │
+│               │                                          │                             │
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)